src/services/backend.js

```diff
@@ -8,6 +8,6 @@ if (!Backend) {
     'module specified by CONNECTOR_BACKEND was not found in /backends')
 }
 
-module.exports = new Backend({
+module.exports = new Backend(config, {
```

Comment:
This makes unit tests cleaner too, because it doesn't require mutating the environment and loading the config